pkg/cmd/step/helm/step_helm_apply.go

```diff
@@ -546,3 +500,75 @@ func (o *StepHelmApplyOptions) fetchSecretFilesFromVault(dir string, store confi
 	}
 	return files, nil
 }
+
+// Sets the Release name
+func setReleaseName(ns string, args []string, options *StepHelmApplyOptions) (releaseName string, err error) {
```

Comment:
It would be nice to add a unit test for this function.

Reply:
Can we please add that in another PR?

Reply:
I think the idea is that the unit test proves the correctness of the refactoring. Unit test really should be part of the production code changes.